data-es2016plus.js

```diff
@@ -6472,7 +6472,7 @@ exports.tests = [
         */},
         res: {
           safari15: true,
-          chrom100: true,
```

Comment:
We should probably throw on unknown browsers in `res`.

Reply:
sounds like a great improvement